geoh5py/ui_json/validation.py

```diff
@@ -133,8 +135,11 @@ def _validations_from_uijson(ui_json: dict[str, Any]) -> dict[str, dict]:
                 validations[key] = {
                     "types": [str, UUID, int, float, Entity],
                 }
-                validations[key]["association"] = item["parent"]
-                validations[key]["uuid"] = None
+                try:
+                    validations[key]["association"] = item["parent"]
+                    validations[key]["uuid"] = None
+                except KeyError:
```

Comment:
The try except is a bit patchy. 
Why do you include it? Should we raise a specifi warning?

Reply:
That's unrelated to this issue, but pylint just caught it. It's possible that the validation just doesn't exist, so we skip over. To be removed soon anyway, just need the linters to pass.

Reply:
For pylint whims, should we do this kind of code or just shut them off at the corresponding line?

Reply:
Agree. Feel free to revert it on your PR, and just add the disable instead. I am cool with that.

Reply:
So I accept and do the changes on my PR?

Reply:
Ideally yes. So we know who did the changes.